### PR TITLE
LPS-164616 | Add Autom. test FDSCustomView#CanBeSaved

### PR DESCRIPTION
**Pull request by eltonccdantas**
**Test Plan**

- Given Custom view enabled for a dataset display instance
- When Make some changes to dataset display
- And When Save As the custom view
- Then A new custom view is created and selected as current view

JIRA Ticket:
https://issues.liferay.com/browse/LPS-164616

### CODE REVIEW

**Comment by eltonccdantas** [comment]
ci:test:sf

**Comment by eltonccdantas** [comment]
ci:test:frontend:nocompile

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase [hunk 1]
```diff
@@ -49,25 +49,60 @@ definition {
...
-		// To-do CanBeSaved Autom. test
+			Click(locator1 = "FrontendDataSet#SELECT_CUSTOM_VIEW");
 
+			Click(
+				key_itemName = "Save View",
+				locator1 = "FrontendDataSet#SELECT_OPTION");
+		}
+
+		task ("Then A new custom view is created and selected as current view") {
+			FDSCustomView.selectView(key_itemName = "Default View");
```
Comment (john-co) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase:89]: Add `ObjectCustomViews.assertColumnNotPresent(columnName = "Description");`

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path [hunk 2]
```diff
@@ -168,6 +168,41 @@
...
+	<td>//button[contains(@class,'dropdown-item') and contains(text(), '${key_itemName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>INPUT_CUSTOM_VIEW_NAME</td>
+	<td>//input[contains(@id,'customViewLabelInput')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SAVE_BTN</td>
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path:197]: Change to SAVE_BUTTON

**Review by john-co — APPROVED** [review]
Looks good to me! :sunglasses:

**Review by mateusralv — APPROVED** [review]

**Comment by john-co** [comment]
```
     [java] java.lang.Exception: Found formatting issues:
     [java] At least one commit message is missing a reference to a required JIRA project: BLADE, CLDSVCS, COMMERCE, FACES, IDE, LCD, LHC, LOOP, LPS, LRAC, LRCI, LRDOCS, LRIS, LRQA, OAUTH2, POSHI, RELEASE, SYNC, TR. Please verify that the JIRA project keys are specifiedin ci.properties in the liferay-portal repository
```

You'll want to update one of your commits to start with LPS-***

**Pull request closed without merging**